cli/inagoctl.go

```diff
@@ -82,6 +97,12 @@ func init() {
 	MainCmd.PersistentFlags().BoolVar(&globalFlags.NoBlock, "no-block", false, "block on synchronous actions")
 	MainCmd.PersistentFlags().BoolVarP(&globalFlags.Verbose, "verbose", "v", false, "verbose output")
 
+	MainCmd.PersistentFlags().StringVar(&globalFlags.Tunnel, "tunnel", "", "use a tunnel to communicate with fleet")
+	MainCmd.PersistentFlags().StringVar(&globalFlags.SSHUsername, "ssh-username", "core", "username to use when connecting to CoreOS machine")
+	MainCmd.PersistentFlags().DurationVar(&globalFlags.SSHTimeout, "ssh-timeout", time.Duration(10*time.Second), "timeout in seconds when establishing the connection via SSH")
+	MainCmd.PersistentFlags().BoolVar(&globalFlags.StrictHostKeyChecking, "strict-host-key-checking", true, "verify host keys presented by remote machines before initiating SSH connections")
+	MainCmd.PersistentFlags().StringVar(&globalFlags.KnownHostsFile, "known-hosts-file", "~/.fleetctl/known_hosts", "file used to store remote machine fingerprints")
```

Comment:
True, but I'd like `tunnel` to be just called `tunnel`, as it is shorter and fleetctl also just calles it tunnel.